managed/CounterStrikeSharp.API/Modules/Menu/ChatMenu.cs

```diff
@@ -45,33 +48,20 @@ public override void Display()
                 var option = Menu.MenuOptions[i];
 
                 Player.PrintToChat(
-                    $" {(option.Disabled ? ChatColors.Grey : ChatColors.Green)} !{keyOffset++} {ChatColors.Default}{option.Text}");
+                    $" {(option.Disabled ? ChatColors.Grey : ChatColors.Green)} [!{keyOffset++}] {ChatColors.Default}{option.Text}");

```

Comment:
Changing the value that comes before the option text would limit our option display characters even further, I don't think this is a good change.